codecs/opus/src/main/jni/opus_jni.cpp

```diff
@@ -22,17 +22,44 @@
 *
 */
 
+#define OPUS_EXPORT __declspec(dllimport)
+#include "opus.h"
 
-#include "opus_jni.h"
+#include "jni.h"
 
-#include "opus.h"
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FRAME_SIZE 480
```

Comment:
Open ticket to make these variables configurable in the future.

Reply:
The hardcoded values will be substituted by configurable values in task #551